src/components/EnvironmentVariables/index.js

```diff
@@ -0,0 +1,427 @@
+                          <Collapse in={openPrjVars}>
+                            <td className="varValue" id={index}>
+                              {projEnvVar.value.length <= 100 &&
+                              !prjValueState[index] ? (
+                                <div className="showHideContainer">
+                                  {hashValue(projEnvVar.value).substring(0, 25)}
+                                  <a
+                                    href="#"
+                                    onClick={() => prjValuesShow(index)}
+                                  >
+                                    <Image
+                                      src={show}
+                                      className="showHide"
+                                      style={{ all: "unset" }}
+                                      alt=""
+                                    />
+                                  </a>
+                                </div>
+                              ) : projEnvVar.value.length <= 100 &&
+                                prjValueState[index] ? (
+                                <div className="showHideContainer">
+                                  {projEnvVar.value}
+                                  <a
+                                    href="#"
+                                    onClick={() => prjValuesHide(index)}
+                                  >
+                                    <Image
+                                      src={hide}
+                                      className="showHide"
+                                      style={{ all: "unset" }}
+                                      alt=""
+                                    />
+                                  </a>
+                                </div>
+                              ) : projEnvVar.value.length >= 100 &&
+                                !prjValueState[index] ? (
+                                <div className="showHideContainer">
+                                  {hashValue(projEnvVar.value).substring(0, 25)}
+                                  <a
+                                    href="#"
+                                    onClick={() => prjValuesShow(index)}
+                                  >
+                                    <Image
+                                      src={show}
+                                      className="showHide"
+                                      style={{ all: "unset" }}
+                                      alt=""
+                                    />
+                                  </a>
+                                </div>
+                              ) : projEnvVar.value.length >= 100 &&
+                                prjValueState[index] ? (
+                                <div className="showHideContainer">
+                                  ${projEnvVar.value.substring(0, 25)}..
+                                  <a
```

Comment:
same with using spans instead or you could actually add the click handler to the image